### ISSUE TITLE
doesn't remove whitespace

### ISSUE THREAD

**Issue by hanskoff** [gruntjs/grunt-contrib-htmlmin#117]
Hi, your plugin is awesome and I'm a happy user of it, but I see it doesn't remove crucial whitespace for me :(

I mean from
```
<div class="form-wrapper">
          <input type="email" name="EMAIL" placeholder="Enter your email">
          <input type="hidden" value="test1234434" name="UNIQUECODE" class="" id="mce-UNIQUECODE">
          <button>Get Early Access</button>
        </div>
```
it minifies to
```
<div class=form-wrapper><input type=email name=EMAIL placeholder=\"Enter your email\"> <input type=hidden value=test1234434 name=UNIQUECODE id=mce-UNIQUECODE> <button>Get Early Access</button></div>
```

as you can see there are `> <` left between elements, what is not what I've expected. Is it a matter of option (I've tried all of them) or an issue?

**Comment by boussouira** [comment]
Are you using `conservativeCollapse` option? you should use `collapseWhitespace`

**Comment by hanskoff** [comment]
Yes, I'm using collapseWhitespace directive. I've moved to jade where there
is no such a problem

Closed #117 <https://github.com/gruntjs/grunt-contrib-htmlmin/issues/117>.

—
Reply to this email directly or view it on GitHub
<https://github.com/gruntjs/grunt-contrib-htmlmin/issues/117#event-542033673>
.
